driver/network_driver.go

```diff
@@ -71,6 +71,14 @@ func (d NetworkDriver) CreateNetwork(request *network.CreateNetworkRequest) erro
 		}
 	}
 
+	// Calico driver does not allow you use the --internal flag
+	internal, ok := request.Options["com.docker.network.internal"].(bool)
```

Comment:
I wonder if we should have a catch-all for any unexpected options - or at the very least be logging any options that we don't recognize?